projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cpq/quote-download.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as authentication from '../../auth-forms';
+
+export const READ_QUOTE = '@READ_QUOTE';
```

Comment:
Please, move all methods  to projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-quote.ts file.
I think it does not make sense to create an additional quote file to click on the **download proposal** button.